trunk-recorder/call_concluder/call_concluder.cc

```diff
@@ -225,6 +237,19 @@ Call_Data_t Call_Concluder::create_call_data(Call *call, System *sys, Config con
   call_info.call_log = sys->get_call_log();
   call_info.call_num = call->get_call_num();
   call_info.compress_wav = sys->get_compress_wav();
+  
+  std::vector<std::map<int,std::time_t>> current_patches = sys->get_talkgroup_patches();
+  //Loop through each patch patch list.  If our the talkgroup from this call is part
+  //of a patch, add the patch data to the call info
+  BOOST_FOREACH (auto& patch, current_patches) {
+    if (patch.find(call_info.talkgroup) != patch.end()) {
+      //current TG is part of this patch
+      //for each TGID/element in this patch:
+      BOOST_FOREACH(auto& patch_element, patch){
+        call_info.patched_talkgroups.push_back(patch_element.first);
+      }
+    }
+  }
```

Comment:
If I'm following you, it would result in something like this in call_concluder.cc instead?
`call_info.patched_talkgroups = sys->get_talkgroup_patches(call_info.talkgroup);`

If that's what you're thinking, yeah, I can do that easily enough.

Reply:
Yea - having it there, might be helpful as we start to check patches in other parts of the program.